lib/cli/index.js

```diff
@@ -77,6 +65,8 @@ function configureLogger(env) {
 
 // Options: Array cliArgs, Stream inputStream, Stream outputStream
 module.exports = function(options) {
+  willInterruptProcess.capture(options.process || process);
```

Comment:
The only thing I'm not aware of is what can happen if we move `capture`/`caprure-exit` from the top of the file inside this function invocation.